layers/+lang/php/funcs.el

```diff
@@ -0,0 +1,42 @@
+;;; funcs.el --- PHP Layer functions File for Spacemacs
+;;
+;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+
+;; lsp
+
+(defun spacemacs//php-setup-backend ()
+  "Conditionally setup php backend."
+  (pcase php-backend
+    (`lsp (spacemacs//php-setup-lsp))))
+
+(defun spacemacs//php-setup-lsp ()
+  "Setup lsp backend."
+  (if (configuration-layer/layer-used-p 'lsp)
+      (lsp)
+    (message "`lsp' layer is not installed, please add `lsp' layer to your dotfile.")))
+
+(defun spacemacs//php-setup-company ()
+  "Conditionally setup company based on backend."
+  (message "%s setting up company" php-backend)
+  (pcase php-backend
+    (`lsp (spacemacs//php-setup-lsp-company))))
+
+(defun spacemacs//php-setup-lsp-company ()
```

Comment:
`lsp` automatically configures company if it is present so this is not needed. Not sure whether the other layers are updated.

Reply:
I now removed the company-related stuff, how does it look to you now?

Please also note, that I'm explicitly keeping the commits here for documentation purposes. I'm going to squash these once this PR is ready to be cherry-picked.